UnityProject/Assets/ScriptableObjects/Learning/Tips.asset

```diff
@@ -0,0 +1,131 @@
+    never know what's on the other side.
+  - Flashlights emit light even when in your pocket. You don't have to hold them
+    to light your way unlike other games.
+  - Face huggers cannot lock themselves on your face if you have a mask on.
+  - Running makes you hungry faster, so does wearing a hardsuit.
+  - Hardsuits slow you down.
+  - In space, no one can hear you scream.
+  - You can use crowbars to open fire doors and unpowered doors.
+  - There's always a traitor among us. Never trust anyone.
+  - Sometimes it's best to just sit back and relax at the bar or make friendships
+    while playing cards or Dungeons and Dragons in the dorms.
+  - You cannot preform some actions while you're handcuffed or missing a limb.
+  - There are four types of damage your character can receive; brute, burn, toxin
+    and oxygen. All of them are equally deadly but have different effects on you.
+  - Boxes automatically pick minnerals from the ground when pulled.
+  - Never take pills labaled as "Happy Pills".
+  - There are over a hunrded recipes for drinks and food. Try and learn them all.
+  - ProtoKineticAccelators (PKA) can destroy walls.
+  - A crewmate who's competent enough to do their job is an antagonist's worst nightmare.
+  - Remember to do your paperwork before requesting something from the <color=blue>Head
+    of Personnel (HoP)</color>
+  - Preform CPR to revive a dying crewmate by changing your intent to <color=green>Help</color>
+    then targeting the best.
+  - Snacks from vending machines aren't fuffling, but they're good enough to keep
+    you running for a couple more minutes.
+  - Say <i>"Come Ian!"</i> near Ian the dog to make him come to you.
+  - Drinking alchole too much gives you liver damage. Which means an annoying visit
+    to medbay if you want to continue living.
+  - Your character's age has minimal effects on gameplay. Don't worry about being
+    too young or too old.
+  - Buckle people and yourself into chairs by dragging their character to the chair.
+  - A handcuffed or restrainted player cannot move freely if you grab or pull them.
+  - Cat ears change the way you speak. Like Thwise, Mwaster~ uwu.
+  - Customization options don't end in the character creator. Visit a vending machine
+    that sells costumes, outfits and accesories to make your character look the way
+    you'd love.
+  - You can be sucked into space if you're not careful enough with your steps and
+    where you go.
+  - If you see purple glitter spreading across the hallway or a room. Run.
+  - Never trust monkeys.
+  - Attacking someone for no reason is the fastest way to get banned off a server.
+  - If you die, there's always a chance you can respawn as another character or play
+    as a "Ghost Role".
+  - "You can restore your hygiene by washing your hands in a sink or taking a shower
+    \u2014 click the showerhead to turn it off and stand under it for a bit."
+  - Some computers and doors require an access level to be used.
```

Comment:
```suggestion
  - Some doors and computers require an access card to be used.
```

Reply:
We still need to tell them about access levels.

Reply:
maybe something like: "id card with proper access level"?